### ISSUE TITLE
nightly test 5-5 failed to deploy vcenter server on nimbus

### ISSUE THREAD

**Issue by chengwang86** [vmware/vic#5536]
Seen on vsphere 6.5

[5-5-Heterogeneous-ESXi.zip](https://github.com/vmware/vic/files/1089102/5-5-Heterogeneous-ESXi.zip)

Failed at `Deploy Nimbus vCenter Server Async ${vc}`

```
2017-06-19 19:14:09 -0700: /mts/git/bin/../nimbus/lib/nimbus/deploys/vc/vcva_deploy.rb:2436:in `<main>'
2017-06-19 19:14:09 -0700: NimbusVcDeployError: VC Deploy Error: VCVA failed to power on.
2017-06-19 19:14:09 -0700: /mts/git/bin/../nimbus/lib/nimbus/deploys/vc/vcva_deploy.rb:639:in `rescue in _deploy'
2017-06-19 19:14:09 -0700: /mts/git/bin/../nimbus/lib/nimbus/deploys/vc/vcva_deploy.rb:589:in `_deploy'
2017-06-19 19:14:09 -0700: /mts/git/nimbus/lib/nimbus/base_deploy.rb:620:in `run'
2017-06-19 19:14:09 -0700: /mts/git/nimbus/lib/nimbus/base_deploy.rb:550:in `block in run_cli'
2017-06-19 19:14:09 -0700: /mts/git/nimbus-toolchain/lin64/ruby/2.2.0/lib/ruby/2.2.0/timeout.rb:89:in `block in timeout'
2017-06-19 19:14:09 -0700: /mts/git/nimbus-toolchain/lin64/ruby/2.2.0/lib/ruby/2.2.0/timeout.rb:34:in `block in catch'
2017-06-19 19:14:09 -0700: /mts/git/nimbus-toolchain/lin64/ruby/2.2.0/lib/ruby/2.2.0/timeout.rb:34:in `catch'
2017-06-19 19:14:09 -0700: /mts/git/nimbus-toolchain/lin64/ruby/2.2.0/lib/ruby/2.2.0/timeout.rb:34:in `catch'
2017-06-19 19:14:09 -0700: /mts/git/nimbus-toolchain/lin64/ruby/2.2.0/lib/ruby/2.2.0/timeout.rb:104:in `timeout'
2017-06-19 19:14:09 -0700: /mts/git/nimbus-toolchain/lin64/ruby/2.2.0/lib/ruby/2.2.0/timeout.rb:125:in `timeout'
2017-06-19 19:14:09 -0700: /mts/git/nimbus/lib/nimbus/base_deploy.rb:550:in `run_cli'
2017-06-19 19:14:09 -0700: /mts/git/bin/../nimbus/lib/nimbus/deploys/vc/vcva_deploy.rb:2436:in `<main>'
2017-06-19 19:14:09 -0700: Writing deployment results...
2017-06-19 19:14:09 -0700: Cloudvm deployment failed. Status: infra_error Phase: Configuring base VM
```

Seems like a timeout issue.

**Comment by mhagen-vmware** [comment]
https://bugzilla.eng.vmware.com/show_bug.cgi?id=1928024

**Comment by mhagen-vmware** [comment]
moving to not ready, this needs a response from nimbus team before proceeding.

**Comment by mhagen-vmware** [comment]
sadly, does not look like nimbus is going to do anything about this - so going to put in another retry

**Comment by jzt** [comment]
[5-2-Cluster.zip](https://github.com/vmware/vic/files/1360334/5-2-Cluster.zip)

**Comment by mhagen-vmware** [comment]
fixed with #6530
